2.ML_model/0.split_data.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Split feature data
+# ## Create tsv file with indexes for held out data, training data, and testing data
+# ### Import libraries
+
+# In[1]:
+
+
+import pandas as pd
+import numpy as np
+import pathlib
+from typing import Tuple, Any, List, Union
+
+from sklearn.utils import shuffle
+
+from utils.MlPipelineUtils import get_features_data, get_random_images_indexes, get_representative_images, get_image_indexes
+
+
+# ### Load data and set holdout/test parameters
+
+# In[2]:
```

Comment:
There is a really important step that you are missing:

setting a random seed

```python
np.random.seed(42)  # Or any number
```

This ensures reproducibility in data splitting.